flink-cdc-common/src/main/java/org/apache/flink/cdc/common/sink/DataSink.java

```diff
@@ -30,5 +33,5 @@ public interface DataSink {
     EventSinkProvider getEventSinkProvider();
 
     /** Get the {@link MetadataApplier} for applying metadata changes to external systems. */
-    MetadataApplier getMetadataApplier();
+    MetadataApplier getMetadataApplier(Set<SchemaChangeEventType> enabledEventTypes);
```

Comment:
This is a public API, we should keep the backward compatibility

Reply:
Done, moved `setAcceptedSchemaEvolutionTypes` to `MetadataApplier` interface, so `DataSink` could be left unchanged.